spec/dd-agent_spec.rb

```diff
@@ -205,6 +205,122 @@ def set_env_var(name, value)
     it_behaves_like 'version set below 4.x'
   end
 
+  context 'allows a string for agent version' do
+    cached(:chef_run) do
+      ChefSpec::SoloRunner.new(
+        :platform => 'ubuntu',
+        :version => '14.10'
+      ) do |node|
+        node.set['datadog'] = {
+          'api_key' => 'somethingnotnil',
+          'agent_version' => '4.4.0-200'
```

Comment:
nit: could you use a version with an epoch here since the platform is ubuntu? (`1:5.9.0-1`)